Jenkinsfile.download_all_logs

```diff
@@ -0,0 +1,45 @@
+pipeline {
```

Comment:
I don't think we should create another pipeline

Reply:
there is a different pipeline, 
renamed the old one to download_failed_cluster_logs and added Jenkinsfile.download_all_logs

Reply:
Why do we need two pipelines? It can be the same one with different parameters.